### DISCUSSION TITLE
How could I reset the password for uptime kuma

### DISCUSSION THREAD

**Discussion by umtauscher** [hassio-addons/addon-uptime-kuma#115] [Q&A]
Hi,
unfortunately I have forgotten the password for the uptime kuma plugin and now I am unable to change anything.
I am running the addon on homeassistant OS.
Is there any way to reset the password/user?
TIA

**Comment by ha01-r36** [comment]
If you can access a terminal such as the one `Advanced SSH & Web Terminal` offers.
You can find the name of your uptime-kuma container with this command:
`docker ps -a --format "{{.Names}}" --filter name=uptime-kuma`

Insert container name in this command:
`docker exec -it [nameFromAbove] npm run --prefix uptime-kuma reset-password -- --data-dir=/data/`

You should end up with something that looks like this:
`docker exec -it addon_a0d7b954_uptime-kuma npm run --prefix uptime-kuma reset-password -- --data-dir=/data/`

Run the command (taking note of the username in case you forgot it) and then enter your new password twice.
